src/resources/postcss/forms.pcss

```diff
@@ -147,4 +149,39 @@
 		}
 	}
 
+	.tribe-tickets__form-message {
+		display: flex;
+		align-items: flex-start;
+		background-color: var(--color-accent-primary-background);
+		border-radius: var(--border-radius-default);
+		padding: var(--spacer-4) var(--spacer-3);
+		margin-bottom: var(--spacer-4);
+
+		@media(--viewport-medium) {
+			padding: var(--spacer-3);
+		}
+	}
+
+	.tribe-tickets__form-message--error {
+		background-color: var(--color-background-error);
+	}
+
+	.tribe-tickets__form-message--error-icon {
+		display: inline-block;
+		margin-right: var(--spacer-2);
+
+		circle {
+			stroke: var(--color-icon-active);
+		}
+
+		circle#dot {
+			stroke: transparent;
+			fill: var(--color-icon-active);
+		}
+
+		path {
+			stroke: var(--color-icon-active);
+		}
+	}
```

Comment:
can probably remove these right? is there any other `.tribe-tickets__form-message` used in the application? or can that above be removed too?

Reply:
I'm currently implementing the ARI variation, so I'm not sure if I'll use them. At the same time is part of the design system and I'm trying to componetize the whole "ET form system". That's why I kept them.